src/hooks/hooks.tsx

```diff
@@ -267,15 +240,15 @@ export const parseJiraDescription = (description: ADFEntity) => {
         return <br />;
       case "inlineCard":
         return (
-          <Link href={node.attrs?.url} target="_blank">
+          <Link href={node.attrs?.url as string} target="_blank">
```

Comment:
This doesn't make sense, optionally chain but typecast as string. This surely means the value can be a string or undefined.

Reply:
The node is a `ADFEntity` type from `@atlaskit/adf-utils` and it is equal to `attrs?: { [name: string]: any }`, so I added a check `url` on the line above and assert that it is definitely a `string`, not just `any`